doc/classes/Path2D.xml

```diff
@@ -13,5 +13,14 @@
 		<member name="curve" type="Curve2D" setter="set_curve" getter="get_curve">
 			A [Curve2D] describing the path.
 		</member>
+		<member name="debug_custom_color" type="Color" setter="set_debug_custom_color" getter="get_debug_custom_color" default="Color(1, 1, 1, 1)">
+			The custom color used to draw the path debug when [member debug_custom_enabled] is enabled.
+		</member>
+		<member name="debug_custom_enabled" type="bool" setter="set_debug_custom_enabled" getter="get_debug_custom_enabled" default="false">
+			If [code]true[/code], the debug custom properties override the global defaults.
+		</member>
+		<member name="debug_enabled" type="bool" setter="set_debug_enabled" getter="get_debug_enabled" default="true">
```

Comment:
Isn't exposing `debug_enabled` a bit redundant, since you could set `debug_custom_color` to a fully transparent colour to essentially get rid of the visualization?